internal/util/update/resource-merge.go

```diff
@@ -224,7 +224,7 @@ func getSubDirsAndNonKrmFiles(root string) (sets.String, sets.String, error) {
 			}
 			return nil
 		}
-		match, err := r.ShouldSkipFile(info)
+		match, err := r.ShouldSkipFile(info.Name(), &kio.IgnoreFilesMatcher{})
```

Comment:
I don't think this is the right way to fix this. I have a fix for this in https://github.com/GoogleContainerTools/kpt/pull/996.

Reply:
Rebased.